config/database.yml

```diff
@@ -12,10 +12,20 @@ default: &default
 development:
   <<: *default
   database: <%= ENV['DB_NAME'] || 'registration_api_development' %>
+  username: root
+  password: rootuser
+  pool: 5
+  host: 127.0.0.1
+  port: 3306
```

Comment:
Don't override the defaults here. You'll break everyone else's development environments and the CI.
If you need different values for your local either make the change and don't commit it or just use the env vars.